lib/rules/no-builtin-form-components.js

```diff
@@ -0,0 +1,25 @@
+import Rule from './_base.js';
+
+const WHY = 'Built-in form components use two-way binding to mutate values.';
+const ACTION = 'Instead, refactor to use a native HTML element.';
+export const MESSAGES = {
+  Input: `Do not use the \`Input\` component. ${WHY} ${ACTION}`,
+  Textarea: `Do not use the \`Textarea\` component. ${WHY} ${ACTION}`,
+};
+
+const COMPONENTS = new Set(['Input', 'Textarea']);
+
+export default class NoBuiltinFormComponents extends Rule {
+  visitor() {
+    return {
+      ElementNode(node) {
```

Comment:
Does this work?
All glimmer Ast should need prefix `Glimmer`.
So `GlimmerElementNode`